mappings/net/minecraft/server/world/chunk/light/ServerLightingProvider.mapping

```diff
@@ -1,2 +1,7 @@
 CLASS uu net/minecraft/server/world/chunk/light/ServerLightingProvider
+	METHOD a doLightUpdates (IZZ)I
```

Comment:
I would say updateLighting would make slightly more sense.

Reply:
I think this would make more sense if this was updating lighting at specified position. This is processing all the scheduled light updates.